docs/services/remove-service.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Add a service
+intro: Add services to servers and applications so they can easily be managed in one place.
```

Comment:
@pvdevs, you forgot to update this information, because this is the documentation for 'remove service', and the information is about 'add service'. Could you fix this?

Reply:
Thank you for catching that Diego, I’ll make the necessary changes. Sorry for the oversight

Reply:
@Diegiwg Addressed on commit 4be0c1b82ffbe88dd1e661a6743bdb32391a11bc